src/config.ts

```diff
@@ -1,6 +1,6 @@
 const config = {
-  REDIRECT_URI: 'REPLACE_WITH_YOUR_REDIRECT_URI', // i.e http://localhost:3000, https://serverName/lf-sample/index.html
-  CLIENT_ID: 'REPLACE_WITH_YOUR_CLIENT_ID',
+  REDIRECT_URI: 'http://localhost/lf-sample-app-react', // i.e http://localhost:3000, https://serverName/lf-sample/index.html
```

Comment:
revert this
try not to commit this file at all

Reply:
oops, forgot to revert on this repo